addons/respawn/XEH_postInit.sqf

```diff
@@ -1,5 +1,4 @@
-// by commy2
 #include "script_component.hpp"
 
 ["rallypointMoved", {_this call FUNC(updateRallypoint)}] call EFUNC(common,addEventhandler);
-["playerChanged", {_this call FUNC(handlePlayerChanged)}] call EFUNC(common,addEventhandler);  // hide enemy rallypoint markers
+["playerChanged", {_this call FUNC(handlePlayerChanged)}] call EFUNC(common,addEventhandler); // hide enemy rallypoint markers
```

Comment:
Yes both are of type `CODE`.
But if you take a look at 
https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_addEventHandler.sqf#L31
you'll notice that this will not reference the code, but actually make a copy of it.